eclass/ruby-ng.eclass

```diff
@@ -318,8 +320,8 @@ ruby_get_use_implementations() {
 ruby_get_use_targets() {
 	debug-print-function ${FUNCNAME} "${@}"
 
-
-	local impls="$(_ruby_get_all_impls)"
+	_ruby_get_all_impls
```

Comment:
I'm confused why you call it explicitly here (but only here) and in outer scope.

Reply:
That looks like a rebase fail I fixed in a later commit.